test/util/fixtures/ClientServiceFixtures.java

```diff
@@ -0,0 +1,32 @@
+package util.fixtures;
+
+import entities.Client;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+
+public final class ClientServiceFixtures {
+  public static final Long FAKEID = 10L;
+
+  public static List<Client> buildClients(Integer size) {
+    return LongStream.range(0, size)
+        .mapToObj(ClientServiceFixtures::buildClient)
+        .collect(Collectors.toList());
+  }
+
+  public static Client buildClient() {
+    return buildClient((Client) null);
+  }
+
+  public static Client buildClient(Long id) {
+    var clientExample = new Client();
+    clientExample.setId(id);
+    return buildClient(clientExample);
+  }
+
+  public static Client buildClient(Client clientExample) {
```

Comment:
Where is the client code being set now?

Reply:
solved, added "testing-code" to buildClient()